code/extensions/che-remote/__mocks__/vscode.ts

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable header/header */
```

Comment:
Why do we need that rule here?

Reply:
we have the rule in every che files else it tries to validate with Microsoft license

Reply:
I see. Thanks for the explanation!

Reply:
but I'll move this line after the header
At first I thought it was only working at the beginning of the file

And I think that probably we could tweak linter rule to ignore the che-* extensions

Reply:
moved the line to the bottom